web/src/app/schedules/temp-sched/TempSchedAddShiftForm.tsx

```diff
@@ -18,6 +19,11 @@ const useStyles = makeStyles({
 export default function TempSchedAddShiftForm(): JSX.Element {
   const classes = useStyles()
   const [manualEntry, setManualEntry] = useState(false)
+  const [zone] = useURLParam('tz', 'local')
+  const now = DateTime.local()
```

Comment:
This is going to change every time it renders, making it possible to set a value that becomes invalid depending on when the user clicks a button.

Maybe this would be a good case for state? Then minimum would basically be set on mount so it would advance through the day (e.g. if user doesn't refresh) but not "break" while they are filling out the form